packages/app-bridge/src/types/Emitter.ts

```diff
@@ -99,6 +99,9 @@ export type EmitterEvents = {
 
     'AppBridge:OpenNavigationManager': void;
 
+    'AppBridge:OpenSearchDialog': void;
+    'AppBridge:CloseSearchDialog': void;
```

Comment:
Should we follow the previous conventions and actionify the event names?
```suggestion
    'AppBridge:SearchDialogOpened': void;
    'AppBridge:SearchDialogClosed': void;
```

Reply:
I would leave it in the imperative form, otherwise it would change the meaning. We don’t want to emit an event to let others know that the search has been opened/closed. We rather want to tell the `web-app` that it should open the search dialog. Same as we did with `AppBridge:OpenNavigationManager` on line 100.